pkg/querier/querier_test.go

```diff
@@ -237,9 +237,11 @@ func TestQuerier_SeriesAPI(t *testing.T) {
 	mockSeriesResponse := func(series []map[string]string) *logproto.SeriesResponse {
 		resp := &logproto.SeriesResponse{}
 		for _, s := range series {
-			resp.Series = append(resp.Series, logproto.SeriesIdentifier{
-				Labels: s,
-			})
+			id := logproto.SeriesIdentifier{}
```

Comment:
We use this pattern to generate the Series in several places. Would it be worth pulling into a util?

Reply:
Done